python/cudf/setup.py

```diff
@@ -44,11 +45,10 @@
             "../../cpp/include/cudf",
             "../../cpp/include",
             "../../cpp/build/include",
-            "../../cpp/build/_deps/cub-src",
-            "../../cpp/build/_deps/thrust-src",
             "../../thirdparty/libcudacxx/include",
             os.path.dirname(sysconfig.get_path("include")),
             np.get_include(),
+            pa.get_include(),
```

Comment:
This points inside the pyarrow install instead of into the arrow-cpp install. The PyArrow install also doesn't include the arrow cuda headers.

Reply:
Aren't we getting the CUDA-enabled pyarrow package as well? It looks like the CUDA headers are in the include paths I'm seeing:

```
$ python
Python 3.7.6 | packaged by conda-forge | (default, Jun  1 2020, 18:57:50) 
[GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyarrow as pa
pa>>> pa.get_include()
'/home/ptaylor/dev/rapids/compose/etc/conda/cuda_10.2/envs/rapids/lib/python3.7/site-packages/pyarrow/include'
>>> 
(rapids) rapids@compose:~$ ls -l /home/ptaylor/dev/rapids/compose/etc/conda/cuda_10.2/envs/rapids/lib/python3.7/site-packages/pyarrow/include/arrow/gpu/cuda_
cuda_api.h        cuda_arrow_ipc.h  cuda_context.h    cuda_memory.h     cuda_version.h    
```

Reply:
I think those are the same ones in `arrow-cpp` that are at the `$PREFIX/include` level. It might be simpler to use those instead of fishing around inside the `pyarrow` package contents.

Reply:
In hindsight I don't think we even need this line since it would be installed at `$PREFIX/include/arrow/gpu` which is already found.

Reply:
Going with this route to better support users not building with conda.